packages/form-js-editor/src/features/properties-panel/PropertiesPanelHeaderProvider.js

```diff
@@ -30,6 +31,10 @@ export const PropertiesPanelHeaderProvider = {
       return textToLabel(field.text);
     }
 
+    if (type === 'image') {
+      return null;
```

Comment:
Perhaps the `altText` would make sense here?

Reply:
![image](https://user-images.githubusercontent.com/17801113/200045788-69584106-5930-4d30-8443-230a7d881012.png)

Reply:
I like that, let's go for it 👍